tgeraser/core.py

```diff
@@ -16,6 +16,8 @@
     -l --limit NUM              Show a specified number of recent chats.
     -t --time-period STRING     Specify a period for an infinite loop to run messages deletion every X seconds/minutes/hours/days/weeks.
                                 Example: --time-period "3*days" OR --time-period "5*seconds"
+    -b --before STRING          Delete messages only older than X seconds/minutes/hours/days/weeks.
```

Comment:
I'd like to use a more descriptive option name, such as `-o --older-than`.
Additionally, the word 'only' is redundant in this context.